lua/lazyvim/plugins/coding.lua

```diff
@@ -54,9 +54,33 @@ return {
             if icons[item.kind] then
               item.kind = icons[item.kind] .. item.kind
             end
-            if entry.context.filetype == "rust" then
-              item.menu = nil
+
+            -- Set 'fixed_width' to false if not provided.
+            local fixed_width = vim.g.cmp_fixed_width or false
+
+            -- Set the fixed completion window width.
+            if fixed_width then
+              vim.o.pumwidth = fixed_width
```

Comment:
I checked the cmp codebase, and afaik this doesn't do anything.
In Neovim `pumwidth` is actually a minimum width for the completion window.
So maybe it is used in cmp when using the native completion window. (which we don't)

Reply:
Strange, if I define `vim.g.cmp_fixed_width = 20` then it does show a much smaller window than the default one. Which is being defined in `pumwidth` for fixed size window. So, what would you like the final approach to be like? Just have a `vim.g.cmp_item_menu_width` and only trim the `item.menu` based on that value if possible?